test/e2e/data/infrastructure-vsphere/cluster-template.yaml

```diff
@@ -0,0 +1,211 @@
+---
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: Cluster
```

Comment:
we have a cli that generates the templates for us, I'd rather run it before `make e2e`, this will avoid having to update things into multiple places

Reply:
@yastij I can squash if changes look good.

Reply:
@sedefsavas - looks good, we might want to remove this file though ?

Reply:
At least CAPZ provider is keeping the generated manifest files for e2e. It may be helpful to see what we are testing.

Reply:
Let's keep it for now. it's not the source of truth so we'll need to update it whenever we change the templates